### PR TITLE
Added basic inventory UI

### PR DESCRIPTION
**Pull request by Kaes3kuch3n**

### CODE REVIEW

**Comment by DavidPgl** [comment]
Add summaries:)

**Comment by thuemmels** [comment]
I have found a minor bug. If you add items to the inventory through the button in the editor, but have not opened the inventory ingame at least once, these items will not appear as buttons in the UI. 
When you open the inventory afterwards and add more items, these previously added items will not be included in the stacks.

**Comment by SamuelllG** [comment]
> 
> 
> I have found a minor bug. If you add items to the inventory through the button in the editor, but have not opened the inventory ingame at least once, these items will not appear as buttons in the UI.
> When you open the inventory afterwards and add more items, these previously added items will not be included in the stacks.

Moving the stuff from InventoryUI's Awake to a constructor and instantiating it in the PlayerController's Start/Awake should fix it, I guess

**Review by SamuelllG — DISMISSED** [review]

**Comment by SamuelllG** [comment]
Code is fine, please fix the bug.

**Comment by Kaes3kuch3n** [comment]
Fixed the bug by loading the script on game start 👍

File: Assets/Scripts/UI/ItemButton.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        public Image Icon => icon;
+        public TextMeshProUGUI NameLabel => nameLabel;
+
+        public int Count
+        {
+            get => _count;
+            set
+            {
+                _count = value;
+                countLabel.text = _count.ToString();
```
Comment (DavidPgl) [inline Assets/Scripts/UI/ItemButton.cs]:
```suggestion
                countLabel.SetText(_count.ToString());
```
Use this instead

File: Assets/Scripts/UI/ItemButton.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        [SerializeField] private Image icon;
+        [SerializeField] private TextMeshProUGUI nameLabel;
+        [SerializeField] private TextMeshProUGUI countLabel;
+
+        private int _count;
+
+        public Image Icon => icon;
+        public TextMeshProUGUI NameLabel => nameLabel;
+
+        public int Count
```
Comment (DavidPgl) [inline Assets/Scripts/UI/ItemButton.cs:21]: Add summary, so we know what it is and that it updates the text automatically.
Other than that:

**Review by DavidPgl — DISMISSED** [review]

**Comment by DavidPgl** [comment]
Merge conflicts @Kaes3kuch3n

**Review by SamuelllG — DISMISSED** [review]

File: Assets/Scripts/Editor/InventoryEditor.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
+using UnityEditor;
+using UnityEngine;
+
+namespace Editor
```
Comment (inkompotato) [inline Assets/Scripts/Editor/InventoryEditor.cs:4]: namespace already exists in animal pack

**Review by SamuelllG — APPROVED** [review]

File: Assets/Scripts/Editor/InventoryEditor.cs (hunk 3 above)
Comment (inkompotato) [reply Assets/Scripts/Editor/InventoryEditor.cs:4]: **Important**: Delete or zip or `mv scripts scripts.disables` the Scripts folder in the LowPolyAnimatedAnimals pack

**Review by inkompotato — APPROVED** [review]

**Pull request merged**